encoders/build.gradle

```diff
@@ -0,0 +1,99 @@
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+apply plugin: 'scala'
+
+compileScala.options.encoding = 'UTF-8'
+// fix scala+java mix to all use compileScala which uses correct dependency order
+sourceSets.main.scala.srcDir 'src/main/java'
+sourceSets.test.scala.srcDirs = ['src/test/java', 'src/test/scala',
+                                 'src/dunit/java', 'src/dunit/scala']
+sourceSets.main.java.srcDirs = []
+sourceSets.test.java.srcDirs = []
+
+dependencies {
+
+  compileOnly 'org.scala-lang:scala-library:' + scalaVersion
+  compileOnly 'org.scala-lang:scala-reflect:' + scalaVersion
+
+  compileOnly("org.apache.spark:spark-hive_${scalaBinaryVersion}:${sparkVersion}")
+  testCompile("org.apache.spark:spark-hive_${scalaBinaryVersion}:${sparkVersion}")
+
+  //compile group: 'io.snappydata', name: 'snappydata-store-core', version: snappyStoreVersion
+  if (new File(rootDir, 'store/build.gradle').exists()) {
+    compile project(':snappy-store:snappydata-store-core')
+  } else {
+    compile group: 'io.snappydata', name: 'snappydata-store-core', version: snappyStoreVersion
+  }
+
+  compile project(":snappy-jdbc_${scalaBinaryVersion}")
+
+  compile("org.parboiled:parboiled_${scalaBinaryVersion}:2.1.4") {
+    exclude(group: 'org.scala-lang', module: 'scala-library')
+    exclude(group: 'org.scala-lang', module: 'scala-reflect')
+    exclude(group: 'org.scala-lang', module: 'scala-compiler')
+  }
+
+  compile 'org.apache.tomcat:tomcat-jdbc:8.5.23'
+  compile group: 'com.zaxxer', name: 'HikariCP', version: '2.7.2'
+
+  testCompile project(':dunit')
+  testCompile 'org.scala-lang:scala-actors:' + scalaVersion
+  testCompile "org.scalatest:scalatest_${scalaBinaryVersion}:${scalatestVersion}"
+
+  compile "org.eclipse.collections:eclipse-collections-api:${eclipseCollectionsVersion}"
+  compile "org.eclipse.collections:eclipse-collections:${eclipseCollectionsVersion}"
+
+  testRuntime files("${projectDir}/../tests/common/src/main/resources")
+}
+
+scalaTest {
+  dependsOn ':cleanScalaTest'
+  doFirst {
+    // cleanup files since scalatest plugin does not honour workingDir yet
+    cleanIntermediateFiles(project.path)
+  }
+  doLast {
+    // cleanup files since scalatest plugin does not honour workingDir yet
+    cleanIntermediateFiles(project.path)
+  }
+}
+
+test.dependsOn ':cleanJUnit'
+
+archivesBaseName = 'snappydata-encoders_' + scalaBinaryVersion
+shadowJar {
+  // avoid conflict with the 0.9.2 version in stock Spark
+  relocate 'org.apache.thrift', 'io.snappydata.org.apache.thrift'
+
+  mergeServiceFiles()
+  exclude 'log4j.properties'
+
+  if (rootProject.hasProperty('enablePublish')) {
+    createdBy = 'SnappyData Build Team'
+  } else {
+    createdBy = System.getProperty('user.name')
+  }
+  manifest {
+    attributes(
+        'Manifest-Version'  : '1.0',
+        'Created-By'        : createdBy,
+        'Title'             : "snappydata-shared1_${scalaBinaryVersion}",
```

Comment:
-shared1?

Reply:
Corrected it. Missed it while performing some testing.